analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ExcludeFromCodeCoverageAttributesNeedJustification.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+[ExcludeFromCodeCoverage] // Noncompliant ^2#23 {{Add a justification.}}
+class Noncompliant
+{
+    [ExcludeFromCodeCoverage()] // Noncompliant
+    void WithBrackets() { }
+
+    [System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage] // Noncompliant
+    void FullyDeclaredNamespace() { }
+
+    [global::System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage] // Noncompliant
+    void GloballyDeclaredNamespace() { }
+
+    [ExcludeFromCodeCoverage] // Noncompliant
+    [CLSCompliant(false)]
+    uint Multiple() { return 0; }
+
+    [ExcludeFromCodeCoverage, CLSCompliant(false)]
+//   ^^^^^^^^^^^^^^^^^^^^^^^
+    uint Combined() { return 0; }
+
+    [ExcludeFromCodeCoverage] // Noncompliant
+    Noncompliant() { }
+
+    [ExcludeFromCodeCoverage] // Noncompliant
+    void Method() { }
+
+    [ExcludeFromCodeCoverage] // Noncompliant
+    event EventHandler Event;
+}
+
+interface IInterface
+{
+    [ExcludeFromCodeCoverage] // Noncompliant
+    void Method();
+}
+
+[ExcludeFromCodeCoverage] // Noncompliant
+struct ProgramStruct
+{
+    [ExcludeFromCodeCoverage] // Noncompliant
+    void Method() { }
+}
```

Comment:
Nice coverage!
Add test cases with `[ExcludeFromCodeCoverage(Justification = null)]` and `[ExcludeFromCodeCoverage(Justification = "")]` and mark them as FNs.

Reply:
@martin-strecker-sonarsource I've made a test that shows that it does not trigger on versions of .NET that not contain this property.

I also added support for null and whitespace justifications.

Reply:
Thanks! Thats even better.